posts/project-pythia-overview.md

```diff
@@ -0,0 +1,70 @@
+---
+author: Max Grover
+date: 2021-7-30
+tags: pythia, training, education
+---
+
+# Project Pythia Portal Overview
+
+Last year, the Project Pythia team was formed to "... provide a public, web-accessible training resource that will help educate current, and aspiring, earth scientists to more effectively use both the Scientific Python Ecosystem and Cloud Computing to make sense of huge volumes of numerical scientific data."
+
+The team is formally a collaboration between three main organizations, although contributors come from thoughout the Python ecosystem:
+
+- The National Center for Atmospheric Research (NCAR)
+- University at Albany
+- Unidata
+
+Last month, the [Project Pythia Team](https://projectpythia.org/index.html#the-project-pythia-team) presented progress on the project at the Earthcube Annual Meeting, the major milestones being the development of the [Foundations Book](https://foundations.projectpythia.org/landing-page.html) and the [Resource Gallery](https://projectpythia.org/gallery.html)
+
+## Header section
+
+The main landing page for Project Pythia is known as the ["Pythia Portal"](https://projectpythia.org/index.html#).
+
+![Pythia Portal](images/pythia_portal.png)
+
+The main page provides an overview of the project, including the motivation and a summary of what the goals of this project are. If you scroll down the page
+
+## Start Learning
+
+The next section is ["Start Learning"](https://projectpythia.org/index.html#start-learning) which is where both the [Foundations Book](https://foundations.projectpythia.org/landing-page.html) and the [Resource Gallery](https://projectpythia.org/gallery.html) can be accessed.
+
+### [Foundations Book](https://foundations.projectpythia.org/landing-page.html)
+
+The [Foundations Book](https://foundations.projectpythia.org/landing-page.html) is one of the major milestones that the Project Pythia team has accomplished thus far. This is the place to go if you are looking for content related to foundational Python for Geoscience materials.
+
+Clicking on the link on the Pythia Portal will take you to the JupyterBook containing the instructional material - this can also be accessed using [foundations.projectpythia.org](https://foundations.projectpythia.org/landing-page.html)
```

Comment:
```suggestion
Clicking on the link on the Pythia Portal will take you to the JupyterBook containing the instructional material - this can also be accessed using [foundations.projectpythia.org](https://foundations.projectpythia.org/landing-page.html).
```